layrry-launcher/src/main/java/org/moditect/layrry/internal/LayersImpl.java

```diff
@@ -258,8 +268,10 @@ private void onDirectoryChange(DirectoryChangeEvent event, Entry<Path, List<Stri
                 return;
             }
 
+            System.out.println("### EVENT: " + event);
```

Comment:
If kept then should be converted to a logging statement.

Reply:
Good catch, that's a left-over. We really need to introduce some logging.